src/App.css

```diff
@@ -1,5 +1,313 @@
+@import url('https://fonts.googleapis.com/css2?family=Montserrat:wght@100;300;400;500;600;700;800&display=swap');
+@import url('https://fonts.googleapis.com/css2?family=Roboto+Slab:wght@100;200;300;400;500;600;700&display=swap');
+
 html {
   margin: 0;
   padding: 0;
   box-sizing: border-box;
+  font-family: 'Montserrat', sans-serif;
+}
+
+body {
+  padding: 9.313rem 3.75rem 7.688rem 6.25rem;
+  background-color: #f5f6fa;
+  height: auto;
+}
+
+.App {
+  margin: 0 2.5rem 4.063rem 0;
+  padding: 0 0 49.625rem;
+  border-radius: 4px;
+  box-shadow: 0 0 60px 0 rgba(0, 0, 0, 0.05);
+  background-color: #fafafa;
+}
+
+.nav {
+  display: flex;
+  align-items: center;
+  height: 5.938rem;
+  margin: 0 0 2.375rem;
+  padding: 1.438rem 1.188rem 1.688rem 6.25rem;
+  background-color: white;
+  border-bottom: 1px solid #e8e8e8;
+}
+
+.cms {
+  height: 2.313rem;
+  margin: 0.375rem 2.938rem 0.125rem 0;
+  font-size: 1.975rem;
+  font-weight: bold;
+  color: #0290ff;
+}
+
+.nav-book {
+  width: 3.688rem;
+  height: 1rem;
+  margin: 1.125rem 2.563rem 0.688rem 0;
+  font-size: 0.813rem;
+  letter-spacing: 1.9px;
+  color: #121212;
+  text-decoration: none;
+}
+
+.nav-category {
+  width: 6.563rem;
+  height: 1rem;
+  opacity: 0.5;
+  font-size: 0.813rem;
+  letter-spacing: 1.9px;
+  color: #121212;
+  text-decoration: none;
+}
+
+.bookCard {
+  display: flex;
+  height: 10.625rem;
+  margin: 2.313rem 6.25rem 0.938rem;
+  padding: 2rem 9.188rem 1.625rem 1.688rem;
+  border-radius: 4px;
+  border:
+    solid 1px
+    #e8e8e8;
+  background-color: #fff;
+}
+
+.bookCategory {
+  width: 4.313rem;
+  height: 1.125rem;
+  margin: 0 1.188rem 0 0;
+  opacity: 0.5;
+  font-size: 0.875rem;
+  font-weight: bold;
+  color: #121212;
+}
+
+.bookTitle {
+  max-width: 12.313rem;
+  height: 1.813rem;
+  margin: 0.188rem 20.188rem 0 0;
+  font-size: 1.375rem;
+  font-weight: bold;
+  letter-spacing: -0.2px;
+  color: #121212;
+  font-family: 'Roboto Slab', serif;
+}
+
+.bookAuthor {
+  width: 5.688rem;
+  height: 1.188rem;
+  margin: 0.25rem 0 0;
+  font-size: 0.875rem;
+  font-weight: 300;
+  color: #4386bf;
+  font-family: 'Roboto Slab', serif;
+}
+
+.comment {
+  width: 4.563rem;
+  height: 1.188rem;
+  font-family: 'Roboto Slab', serif;
+}
+
+.remove {
+  width: 3.375rem;
+  height: 1.188rem;
+  font-family: 'Roboto Slab', serif;
+}
```

Comment:
Great job @As1imwe-Mark 🤝

- [ ] [Optional] As stated in the project requirements, Ensure you do not repeat the same rule for multiple selectors for example `Roboto Slab', serif`. Kindly create a single class and reuse this class where necessary. You may take a look at [this file](https://speckyboy.com/good-bad-css-practices/) for more guidance.